e2e/verifyocsp/responder/standalone/add_cert_to_index.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
If you can, please use `#!/bin/sh` and do a shellcheck for POSIX sh compatibility - we are packaged for alpine where `bash` cannot be counted upon.

I can tidy this up post merge if that's a pain.